recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "unifunc" %}
+{% set version = "1.3.1" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/UniFunc/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f113551ef29d1efc60e68c5796844b71a02d06ecfbe9495eeb1ca80e2e7a2a9f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```

This package looks like it's pure Python, any reason it couldn't be noarch? We can ignore any Windows failures due to missing dependencies:
```suggestion
  noarch: python
```

---

I tried with noarch and it was failing on windows.
see here:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=437046&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b

---

It's failing on Windows because it can't read the `README.md` file with Python's default encoding on Windows ('cp1252'). Either remove the unicode quotes in the document (there are several quotes in there which are the "fancy" quotes) or in `setup.py` call `read_text` explicitly with a utf-8 encoding, like:
```python
read_text(encoding='utf-8')
```

---

ah ok. I  will apply your proposed changes tomorrow
thanks for the help!